### ISSUE TITLE
verdi work list/ -a, does not show older workchains anymore

### ISSUE THREAD

**Issue by broeder-j** [aiidateam/aiida_core#930]
verdi work list from current develop, does not show my old/currently any workchain nodes anymore, which are still in the DB. Was Backcompability broken by the introduction of state?
(can somebody reproduce this, or should I resetup?)
I let you know if I find the reason.

**Comment by sphuber** [comment]
I tried to make the interface more similar to `verdi calculation list` and in the old version of `verdi work list` the `-a` flag simply toggled to show _all_ workchains, which is not only different from what many would expect, but also hugely unusable. I changed it such that by default, only none _sealed_ nodes are shown and by using `-a` also the sealed ones are shown. This should be backwards compatible. Even if some older nodes in your database do not have their sealed attribute set properly, they should still show up, even without using the flag (even though they should be sealed). If you are not seeing anything, there must be some other bug. There was a nasty merge I had to do that involved this command, I may have fucked something up. I will give it a look
